src/addRemove.js

```diff
@@ -0,0 +1,85 @@
+import { addToStorage, getFromStorage } from './updateLocalStorage.js';
```

Comment:
It is always better to have your dependencies in a single folder. I will suggest that you move them into a folder name "modules" for easy assessibility